downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/MolecularFunctionAnnotationBuilder.java

```diff
@@ -101,9 +99,9 @@ private static boolean validCatalystAU(GKInstance activeUnitInst) throws Excepti
 
     /**
      * Checks that the incoming EntitySet instance only has EWAS members
-     * @param entitySetInst --  ActiveUnit/PhysicalEntity of a catalyst that is an EntitySet
-     * @return
-     * @throws Exception
+     * @param entitySetInst -- GKInstance,  ActiveUnit/PhysicalEntity of a catalyst that is an EntitySet
+     * @return -- Boolean, true if the instance only contains EWAS' in its hasMember attribute, false if not.
```

Comment:
Probably better to write the type as "boolean" or maybe "Boolean primitive" so it is clear that a primitive is returned.